packages/frontend/styles/components/Borrow.module.css

```diff
@@ -32,3 +32,9 @@
   opacity: 0.5;
   margin-right: 0.5rem;
 }
+
+.btn {
```

Comment:
Don't you think this should be a `variant` of our button in the theme ? 🤔

Reply:
Just add these styles in `primary` and `gradient` `variants`.
But you removed it in `chris/feat/screen-integration` merge request (https://github.com/Fujicracy/fuji-v2/commit/67c3dcee2bec9f971a2949bfc9d4d413ad5111fc) last week. We can talk more about it directly later.